README.md

```diff
@@ -1,10 +1,28 @@
-# Mock-company-name (TBD)
-
-This is an educational repository for the fictional company Mock-company-name which provides a range of video editing services. This company store all of their services/products in a monorepo managed by [pants](https://www.pantsbuild.org/). The goal of this repository is to expand on the examples provided in the original pants documentation and show pants's key concepts in a centralized location.
+# MVP (Minimum Viable Pants)
 
+Even though [pants](https://www.pantsbuild.org/) provide detailed documentation, it lacks a beginner friendly monorepo where users can interact to explore its key concepts. The goals of this repo are:
```

Comment:
It has a beginner friendly monorepo, id say it doesn't show off some of its most useful functionality. It lacks completeness.